.prettierrc

```diff
@@ -1,11 +1 @@
-{
-  "printWidth": 100,
-  "jsxBracketSameLine": false,
-  "singleQuote": false,
-  "tabWidth": 2,
-  "trailingComma": "es5",
-  "semi": true,
-  "arrowParens": "always",
-  "bracketSpacing": true
-}
-
+{}
```

Comment:
Shouldn't we define what rules we want to enforce?

Reply:
You're right, we sure ensure that Prettier supports our own approach to docs. Let's raise it during the team meeting and see what formatting we want to enforce?